cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,47 @@
+import PageObject from '../PageObject';
+class ArticlePageObject extends PageObject {
+  url = '/#/editor';
+
+  get articleTitleField() {
+    return cy.get('[placeholder="Article Title"]');
+  }
+
+  get articleAboutField() {
+    return cy.get(`[placeholder="What's this article about?"`);
+  }
+
+  get articleBodyField() {
+    return cy.get('[placeholder="Write your article (in markdown)"');
+  }
+
+  get articleTagsField() {
+    return cy.get('[placeholder="Enter tags"');
```

Comment:
There is a missing closing bracket in the selector for `articleTagsField`. It should be `cy.get('[placeholder="Enter tags"]')`.